index.yaml

```diff
@@ -174,6 +174,22 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: QuestionSkillLinkModel
+  properties:
+  - name: skill_id
+  - name: question_id
+
+- kind: QuestionSkillLinkModel
+  properties:
+  - name: skill_id
+  - name: skill_difficulty
+
+- kind: QuestionSkillLinkModel
+  properties:
+  - name: skill_id
+  - name: skill_difficulty
+    direction: desc
```

Comment:
is it a duplicate of line 182-185? (Not sure, if so then maybe removing these changes and running the server will fix?)

Reply:
It's generated automatically. I don't think they are duplicated. Line 187 - 191 has `direction: desc` at the end.

Reply:
I think not, the order is different (note the "direction" key).